rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/scan/RxBleInternalScanResult.java

```diff
@@ -11,14 +11,16 @@
     private final long timestampNanos;
     private final ScanRecord scanRecord;
     private final ScanCallbackType scanCallbackType;
+    private final Boolean isConnectable;
 
     public RxBleInternalScanResult(BluetoothDevice bluetoothDevice, int rssi, long timestampNanos, ScanRecord scanRecord,
-                                   ScanCallbackType scanCallbackType) {
+                                   ScanCallbackType scanCallbackType, Boolean isConnectable) {
```

Comment:
I'm new here and am browsing but thought I'd offer a suggestion. To be more explicit, you could give it an enum instead of a nullable boolean like the following (pardon the kotlin):
```
enum class IsConnectableStatus { LEGACY_UNKNOWN, CONNECTABLE, NOT_CONNECTABLE }
```
Otherwise i'm not sure it's clear that `null` means pre-26.

Reply:
Seems like a good solution.